### DISCUSSION TITLE
JUG Meeting the 17th of May 2021

### DISCUSSION THREAD

**Discussion by Llewellynvdm** [joomengine/Joomla-Component-Builder#734] [Announcements]
## You are all welcome

# Topic
- Front-end Views
- Extension Distributor

<strong>17th of May 2021 <a href="https://doodle.com/poll/vs9nzkcszhwipc7z">see time</a> all welcome at <a href="https://meet.google.com/kxr-nfbv-dww">room</a></strong>

**Comment by marcodings** [comment]
**Llewellyn**
>     * Front-end Views
**Emmanuel Danan**
>     * Extension Distributor

**Comment by saman222** [comment]
hi
is it possible put anywhere recorded meeting's file? we was want join but i and my friends have some individual problem.
is it possible JUG Meeting every 15 days instead of 1 time per month?

**Comment by saman222** [comment]
> ## You are all welcome
> # Topic
> * Front-end Views
> * Extension Distributor
> 
> **17th of May 2021 [see time](https://doodle.com/poll/vs9nzkcszhwipc7z) all welcome at [room](https://meet.google.com/kxr-nfbv-dww)**

hi @Llewellynvdm 
is there any recorded meeting file?

**Comment by cpaschen** [comment]
Just asking again ... did anyone happen to record this?

**Reply by Llewellynvdm** [reply]
Sorry guys it was my turn to record this, but it came under my attention that recording the sessions is not that attractive to some of the possible participants, so I am reconsidering the whole recording idea all together. 

The JUG is first a place for any JCB or extension developer to come and have fun, chat, get to know other developers, and learn. So it is about those who are there in the moment. When recording defeats that objective then recording is out... 

We would like to see more people at the JUG, participating in the conversation and since the current trend is an indication of what recording does... I rather not have any more recordings, but instead encourage you to be there.

**Reply by Llewellynvdm** [reply]
What I will do is try to make special tutorials after the JUG about the topics discussed and post those.

Here is the link to some of the topic at the last JUG:
https://www.youtube.com/watch?v=owpxKie0I7s&list=PLQRGFI8XZ_wunXuo0c4cSd7aXLbV2J-_Q&index=1
https://www.youtube.com/watch?v=KyAKuVLQo0E&list=PLQRGFI8XZ_wsV6PDFXd21B7O0X70JlNd3&index=10
https://www.youtube.com/watch?v=GLcPdLXQxrQ&list=PLQRGFI8XZ_wsV6PDFXd21B7O0X70JlNd3&index=11

**Comment by EvDoorne** [comment]
Agreed. I can’t see the added value in recording and publishing the JUG for me/us.

Met dank en vriendelijke groet,

Masters of Media


Erik van Doorne

Rieteweg 17
8041 AJ   ZWOLLE

***@***.***
+31382222011

https://mastersofmedia.nl <https://mastersofmedia.nl/>

**Comment by cpaschen** [comment]
The main thing that is helpful (at least to me) is to have a recording of the 'presentation' part.
Just wondering if it would be helpful (less work for Llewelyn) if we were to record it live and then just post the 'presentation' part - so that you don't have to spend time re-presenting something twice?

As long as I'm on the meeting I'd be willing to help with that process (I do video editing for a number of projects so I have the tools here to do that).
